bitnami/postgresql/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: postgresql
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/postgresql
-version: 16.0.6
+version: 16.0.7
```

Please bump minor version instead:
```suggestion
version: 16.1.0
```